index.html

```diff
@@ -1,55 +1,247 @@
-            <ul class="nav-menu">
-                <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
-                <li class="nav-item"><a href="#" class="nav-link">About</a></li>
-                <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
-            </ul>
-            <div class="handburger">
-                <span class="bar"></span>
-                <span class="bar"></span>
-                <span class="bar"></span>
-            </div>
-        </nav>
+      <nav class="navbar">
+        <a href="#">
+          <h2 class="nav-title">Welcome</h2>
+        </a>
+        <ul class="nav-menu">
+          <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+        </ul>
+        <div class="handburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
     </header>
 
     <section class="first-section">
-        <div class="container">
-            <h1>Hey there. I’m Daniel</h1>
-            <h1 class="desc">I’m a software developer</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-                <ul class="social">
-                    <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
-                    <li><a href="#"> <img src="./Images//linkedin.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images/medium.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
-                </ul>
-        </div>
+      <div class="container">
+        <h1>Hey there. I’m Daniel <br> <span class="desc">I’m a software developer</span> </h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social">
+          <li>
+            <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//github.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+          </li>
+        </ul> 
+      </div>
```

Comment:
- [ ] Currently, your project seems to not align at the center. The images below are the view on a small screen of width 375px. You can see how some elements are hidden to the right. After inspecting your project, I see `margin-left: 50px` and `margin-right: -50px` on the `div.container` if you remove this, you get some proper alignments. I also see that this element does not need grid but flex instead. 
- [ ] Also, if you remove `padding-left: 50px;` and `padding-right: 50px;` from `<ul class="boxes">` Things become a bit better. 
- [ ] Truth is you have a lot of issues going on on your project, I sincerely think you need an overhaul. However, do not be discouraged, just try to fix them one after the other.


**Toolbar & Headline** | Works section | About Section
----| ---- | ----
![image](https://user-images.githubusercontent.com/84156714/212471638-cf1e8e78-c098-4234-88b8-b7e9166f47ab.png) | ![image](https://user-images.githubusercontent.com/84156714/212471682-9c7efcd7-ec6d-403c-9d27-01cd84598312.png) | ![image](https://user-images.githubusercontent.com/84156714/212471694-7eff3413-9da9-49dc-91d0-c807df209672.png)